packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2639,13 +2642,62 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget? icon;
 
-  /// Text that describes the input field.
+  /// Optional widget that describes the input field.
+  ///
+  /// {@macro flutter.material.inputDecoration.labelText}
```

Comment:
Should this macro be named "..inputDecoration.label"?

Reply:
I agree with you. `inputDecoration.label` sounds more appropriate since this is shared between `label` and `labelText`.